### PR TITLE
[Adding] aviquo DNS

### PR DESCRIPTION
**Pull request by om-raheja**
This will be used to host aviquo, an extracurricular finder for high schoolers. Completely free and open source.

### CODE REVIEW

**Comment by alx-alexpark** [comment]
Hey! Sounds like an interesting project. Is this affiliated with Hack Club? Your subdomain should be under dino.icu instead of hackclub.com.

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2907,3 +2907,8 @@ takeout:
   - ttl: 600
     type: A
     value: 68.183.252.105
+
+aviquo:
+  - ttl: 600
+    type: CNAME
```
Comment (alx-alexpark) [inline hackclub.com.yaml:2913]:
```suggestion
    type: A
```
IPv4 address records should use record type A

**Comment by om-raheja** [comment]
Hey, I switched the CNAME to an A record. The project isn't directly associated with hack club, but I'm a high schooler and on https://toolbox.hackclub.com, it says I can get a free hackclub.com subdomain to host some of my projects. Please let me know if any further verification is needed.

**Comment by om-raheja** [comment]
I changed it to dino.icu, but I would prefer if I could have a .org or .com subdomain instead 😅 It'll do for now though, thank you!

**Comment by om-raheja** [comment]
Anything else I need to add?

File: hackclub.com.yaml [hunk 2]
```diff
@@ -2907,3 +2907,4 @@ takeout:
   - ttl: 600
     type: A
     value: 68.183.252.105
+
```
Comment (Muirrum) [inline hackclub.com.yaml:2910]: delete spurious change

**Review by Muirrum — CHANGES_REQUESTED** [review]

**Comment by Muirrum** [comment]
is this still needed?

**Comment by om-raheja** [comment]
no, thank you On Sep 26, 2024, at 12:19, Cara Salter ***@***.***> wrote:﻿
is this still needed?

**Pull request closed without merging**